lab-taylor/app/controllers/game-controller.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+const angular = require('angular');
+const Monster = require('../model/monster');
+const Item = require('../model/items');
+
+angular.module('adventureGame').controller('GameController', [GameController]);
+
+function GameController() {
+  this.history = ['STATION REPORT -- ALL SYSTEMS DOWN -- GOOD LUCK'];
+
+  this.map = require('../model/map');
+  this.dynamicClass = '';
+  this.player = {
+    name: 'Reptar',
+    hp: 100,
+    damage: 5,
+    xp: 0,
+    location: 'moduleA1',
+    items: []
+  };
+
+  this.gameForm = {
+    direction: 'forward'
+  };
```

Comment:
This doesn't make use of the scope created by the form. You can get rid of this.gameForm here. See my comment on the html also.